docs/flink-getting-started.md

```diff
@@ -634,6 +634,26 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
+## UPSERT mode
+
+Iceberg supports `UPSERT` based on the primary key when writing data into v2 table format. 
+
+Enable the `UPSERT` mode on a table by adding the `write.upsert.enabled` configuration. All writing will be in `UPSERT` mode unless you overwrite it when building the writer:
```

Comment:
> All writing will be in `UPSERT` mode unless you overwrite it when building the writer:

This part is not completed. "building the writer" is not followed by example. I would actually suggest remove this part. Instead, you can follow up with the previous sentence as "Here is an example create table SQL statement with upsert table property".

Reply:
Lead this paragraph with `1. `. e.g.,

```
1. Enable the `UPSERT` mode as as table-level property ..... Here is an example SQL statement to set the table property when creating a table.  It would be applied for all write paths to this table (batch or streaming) unless overwritten by write options as described later.
```